active/InputValidation/InputValidation.md

```diff
@@ -0,0 +1,572 @@
+        Text="{x:Bind   ViewModel.Person.Name, 
+                        UpdateSourceTrigger=PropertyChanged, 
+                        Mode=TwoWay}" 
+         UseSystemValidationVisuals="False"
+        />
+
+<!-- Custom error message template -->
+<ItemsControl
+    ItemsSource="{x:Bind Name.(Validation.Errors), Mode=OneWay}"
+    Foreground="Red">
+
+    <ItemsControl.ItemTemplate>
+        <DataTemplate x:DataType="ValidationError">
+            <StackPanel Orientation="Horizontal" Spacing="5">
+                <Border Background="Red" Margin="3" CornerRadius="10" 
+                        Width="12" Height="12" VerticalAlignment="Top">
+                    <TextBlock Text="!" Foreground="White" 
+                               HorizontalAlignment="Center" VerticalAlignment="Center"/>
+                </Border>
+                <TextBlock Text="{x:Bind ErrorMessage, Mode=OneWay}"
+                           />
+            </StackPanel>
+        </DataTemplate>
+    </ItemsControl.ItemTemplate>
+</ItemsControl>
+```
+
+## Customizing the error template
+
+To modify the styling and content of your error template, use the Validation.ErrorTemplate property. The Validation.ErrorTemplate refers to either the content of the tooltip or the content of the container below the control depending on the validation mode.
+
+If you wish to override the default visuals for errors across all forms elements in your app, override the ValidationErrorTemplate DataTemplate and use in your app.
+
+```xaml
+<ResourceDictionary>
+    <DataTemplate x:Key="ValidationErrorTemplate">
+        <StackPanel Orientation="Horizontal">
+            <Border Background="Red" Margin="3" CornerRadius="10" Width="20" Height="20" VerticalAlignment="Top">
+                <TextBlock Text="!" Foreground="White"
+                HorizontalAlignment="Center" VerticalAlignment="Center"/>
+            </Border>
+            <TextBlock Text="{x:Bind (Validation.Errors)[0].ErrorContent, Mode=OneWay}"
+        </StackPanel>
+    </DataTemplate>
+</ResourceDictionary>
+
+<TextBox Validation.ErrorTemplate="{StaticResource ValidationErrorTemplate}"/>
+```
+
+## Change the resource color for the error and description messages
+Colors for the error and description messages are defined in a standard color/brush pattern in generic.xaml that can be overridden at an app level.
+
+```xaml
+<Application.Resources>
+
+    <!--Color for error messages-->
+    <Color x:Key="SystemErrorColor">#CE3880</Color>
+    <SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Value="SystemErrorColor"/>
+
+    <!--Brush for description messages-->
+    <SolidColorBrush x:Key="SystemControlDescriptionTextForegroundBrush" Value="SystemControlPageTextBaseMediumBrush"/>
+
+</Application.Resources>
+```
+
+## Change description text for validation
+
+Description text provides input to the end user on what format or content is expected from the control when it is not clear from the header and placeholder text.
```

Comment:
This sounds useful even if there isn't an error. Why make it related to validation?

Reply:
Great point! I think it's worth mentioning as part of the overall experience for building a form, but maybe it isn't worth calling out as part of validation